Source/Core/VideoCommon/VertexLoaderARM64.cpp

```diff
@@ -288,7 +288,7 @@ void VertexLoaderARM64::ReadColor(u64 attribute, int format, s32 offset)
 			else if (offset & 3) // Not aligned - unscaled
 				LDUR(scratch2_reg, src_reg, offset);
 			else
-				LDR(INDEX_UNSIGNED, scratch3_reg, src_reg, m_src_ofs);
+				LDR(INDEX_UNSIGNED, scratch3_reg, src_reg, offset);
```

Comment:
What part does this change play in "disable the 6666 format"?
Even if its a copy&paste mistake to be fixed, I'd recommend putting that in a separate commit (otherwise it might get reverted along with the "disable 6666" if it isnt done by hand.